extensions/validation/src/test/java/org/mule/extension/validation/ValidationTestCase.java

```diff
@@ -26,7 +26,6 @@
 
     static final String VALID_URL = "http://localhost:8080";
     static final String INVALID_URL = "here";
-
```

Comment:
the goal is to make it more readable to the human. No formatter tool covers 100% of the cases